packages/beacon-node/src/chain/opPools/aggregatedAttestationPool.ts

```diff
@@ -53,7 +54,10 @@ type AttestationWithScore = {attestation: Attestation; score: number};
 export type AttestationsConsolidation = {
   byCommittee: Map<CommitteeIndex, AttestationNonParticipant>;
   attData: phase0.AttestationData;
-  totalNotSeenCount: number;
+  totalNewSeenEffectiveBalance: number;
+  newSeenAttesters: number;
+  notSeenAttesters: number;
+  committeeSize: number;
```

Comment:
We might need a map for `committeeSize`. See comment below

Reply:
I tracked it in `committeeMembers`